pkg/upgrade/upgrades/alter_table_statistics_avg_size_test.go

```diff
@@ -77,16 +77,16 @@ func TestAlterSystemTableStatisticsTable(t *testing.T) {
 		validationSchemas,
 		false, /* expectExists */
 	)
-	// Run the migration.
-	migrations.Migrate(
+	// Run the upgrade.
+	upgrades.Migrate(
```

Comment:
similar question here - should this be called `Upgrade` or still `Migrate`?

Reply:
Done.